core/src/main/java/hudson/PluginManager.java

```diff
@@ -630,6 +630,13 @@ void considerDetachedPlugin(String shortName) {
         }
     }
 
+    /**
+     * Retrieves the location of the detached plugins in the war
```

Comment:
```suggestion
     * Defines the location of the detached plugins in the WAR.
     * @return by default, {@code /WEB-INF/detached-plugins}
     * @since TODO
```